docs/maui/views/MediaElement.md

```diff
@@ -77,11 +77,65 @@ public class MainActivity : MauiAppCompatActivity
 }
 ```
 
+Edit the `androidManifest.xml` manifest file and add the following.
```

Comment:
Let's highlight that the following items need to be added, then let's show an example of a completed `androidManifest.xml` file. 

This will help devs understand what needs to change. If we only show an example of the completed `androidManifest.xml` file, it's easy for a dev to overlook one small change.

I recommend following the same pattern we established for the Snackbar's `Platform specific initialization` for Windows docs : https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/alerts/snackbar?tabs=windows%2Candroid#1-enable-the-snackbar-usage-with-the-mauiappbuilder

These are the additions I see from the [PR](https://github.com/CommunityToolkit/Maui/pull/1782/files):

```xml
<application android:enableOnBackInvokedCallback="true" 
```

```xml
     <service android:name="CommunityToolkit.Maui.Media.Services" android:exported="false" android:enabled="true" android:foregroundServiceType="mediaPlayback">
       <intent-filter>
         <action android:name="android.intent.action.MEDIA_BUTTON" />
       </intent-filter>
       <intent-filter>
         <action android:name="androidx.media3.session.MediaSessionService"/>
       </intent-filter>
     </service>
```

```xml
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
   <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
   <uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
```

Reply:
These are the required additions to `androidManifest.xml` that I see in the [PR](https://github.com/CommunityToolkit/Maui/pull/1782/files):

```xml
<application android:enableOnBackInvokedCallback="true" 
```

```xml
     <service android:name="CommunityToolkit.Maui.Media.Services" android:exported="false" android:enabled="true" android:foregroundServiceType="mediaPlayback">
       <intent-filter>
         <action android:name="android.intent.action.MEDIA_BUTTON" />
       </intent-filter>
       <intent-filter>
         <action android:name="androidx.media3.session.MediaSessionService"/>
       </intent-filter>
     </service>
```

```xml
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
   <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
   <uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
```

Reply:
Updated as requested.